charts/values.yaml

```diff
@@ -449,10 +450,11 @@ haproxy:
 
 ## Configure Ingress
 
-##################################################################
-#  IMPORTANT NOTE:                                               #
-#  Ingress is only supported starting MarkLogic 11.1 and higher. #
-##################################################################
+#######################################################################
+#  IMPORTANT NOTE:                                                    #
+#  Ingress is only supported starting MarkLogic 11.1 and higher.      #
```

Comment:
Ingress is only supported with MarkLogic 11.1 and later.

Reply:
corrected